app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/NotificationsModule.kt

```diff
@@ -0,0 +1,29 @@
+package co.netguru.baby.monitor.client.application.di
+
+import android.content.Context
+import co.netguru.baby.monitor.client.R
+import co.netguru.baby.monitor.client.common.NotificationHandler
+import co.netguru.baby.monitor.client.feature.babynotification.NotifyBabyEventUseCase
+import co.netguru.baby.monitor.client.feature.firebasenotification.FirebaseNotificationSender
+import dagger.Module
+import dagger.Provides
+
+@Module
+object NotificationsModule {
+    @Provides
+    fun notificationHandler(context: Context) = NotificationHandler(context)
+
+    @Provides
+    fun provideNotifyBabyEventUseCase(
+        notificationSender: FirebaseNotificationSender,
+        context: Context
+    ) =
+        NotifyBabyEventUseCase(
+            notificationSender,
+            mapOf(
+                NotifyBabyEventUseCase.CRY_TITLE_KEY to context.resources.getString(R.string.notification_baby_is_crying_title),
+                NotifyBabyEventUseCase.NOISE_TITLE_KEY to context.resources.getString(R.string.notification_noise_detected_title),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/89ff9ffb603a83b79e177dd00519720c6a8d52ac/app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/NotificationsModule.kt#L25"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>